types/wawoff2/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for wawoff2 1.0
+// Project: https://github.com/fontello/wawoff2#readme
+// Definitions by: Luca Becker <https://github.com/lucavb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function compress(src: ArrayLike<number>): Uint8Array;
+export function decompress(src: ArrayLike<number>): Uint8Array;
```

Comment:
1. Both functions return a Promise
2. The docs say `src` is a `Buffer` or `Uint8Array`. Is `ArrayLike<number>` too loose?

Reply:
Indeed you are right. I relied too much on what my IDE told me. I have updated the PR.